dd-trace-php/hooks/build

```
@@ -28,8 +35,32 @@ if [[ $TMP != $DOCKER_TAG ]]; then
 	docker push ${BASE_IMAGE}_7_php_72
 	docker push ${BASE_IMAGE}_7_php_73
 
-	# This image will be pushed in normal push step
+	# This image will be buiild/pushed in normal step processing
+	#   docker build --build-arg CENTOS_VERSION=6 --build-arg PHP_VERSION=54 -f $DOCKERFILE_PATH -t ${BASE_IMAGE}_6_php_54 .	
 	#	docker push ${BASE_IMAGE}_6_php_54
+
+	echo Retag upstream ZTS images
+	ZTS_BASE_IMAGE=${DOCKER_REPO}:ddtrace_php_zts_
```

Assuming you want `_56` instead of `__56` right?

```suggestion
	ZTS_BASE_IMAGE=${DOCKER_REPO}:ddtrace_php_zts
```